libs/ui-lib/src/lib/pill-options/pill-options.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import PillOptions from './pill-options';
+
+const mockOptions: string[] = ['Yes', 'No', 'Maybe'];
+const mockOnChange: jest.Mock<any, any> = jest.fn();
```

Comment:
The use of any gives a linting warning, you can just remove the type and the test still passes:
```suggestion
const mockOnChange: jest.Mock = jest.fn();
```